front/src/modules/settings/data-model/new-object/components/SettingsNewObjectType.tsx

```diff
@@ -16,56 +17,60 @@ const StyledContainer = styled.div`
   display: flex;
   flex-direction: row;
   gap: ${({ theme }) => theme.spacing(2)};
+  margin-bottom: 32px;
   width: 100%;
 `;
-
+const STANDARD_OBJECT_TYPE = 'Standard';
 export const SettingsNewObjectType = ({
-  selectedType = 'Standard',
+  selectedType = STANDARD_OBJECT_TYPE,
```

Comment:
Defining constants for a type is a pattern we don't have in our current codebase, we prefer using string literals and the IDE will autocomplete.

Reply:
Fixed.

Reply:
Fixed.